pkg/apis/openebs/v1alpha1/blockdeviceclaim_types.go

```diff
@@ -53,8 +53,8 @@ type DeviceClaimRequirements struct {
 }
 
 const (
-	// ResourceCapacity defines the capacity required as v1.Quantity
-	ResourceCapacity v1.ResourceName = "capacity"
+	// ResourceStorage defines the storage required as v1.Quantity
+	ResourceStorage v1.ResourceName = "storage"
```

Comment:
Can we eliminate the use of this completely and just use the constant defined in K8s itself?

Reply:
Was thinking the same ...as we are using the same in maya side.

Reply:
Pushed the changes PTAL